README.md

```diff
@@ -198,6 +198,35 @@ buildctl build ... --exporter=oci --exporter-opt output=path/to/output.tar
 buildctl build ... --exporter=oci > output.tar
 ```
 
+### Exporting/Importing build cache (not image itself)
+
+#### To/From registry
+
+```
+buildctl build ... --cache-export type=registry,ref=localhost:5000/myrepo:buildcache
+buildctl build ... --cache-import type=registry,ref=localhost:5000/myrepo:buildcache
+```
+
+#### To/From local filesystem
+
+```
+buildctl build ... --cache-export type=local,store=path/to/input-dir
+buildctl build ... --cache-import type=local,store=path/to/output-dir
+```
+
+The directory layout conforms to OCI Image Spec v1.0.
+
+#### `--cache-export` options
+* `mode=min` (default): only export layers for the resulting image
+* `mode=max`: export all the layers of all intermediate steps
+* `ref=docker.io/user/image:tag`: reference for `registry` cache exporter
+* `store=path/to/output-dir`: directory for `local` cache exporter
```

Comment:
why isn't `tag` (or name) a supported import/export option for local?

Reply:
can be follow-up?